_docs/concepts/network-and-auth/auth.md

```diff
@@ -67,8 +67,9 @@ Service-to-service communication is tunneled through the client side [Envoy](htt
 
 ### Key management
 
-Istio Auth provides a per-cluster CA (Certificate Authority) to automate key and certificate management. It performs four key operations:
+Istio v0.2 supports services running on both Kubernetes pods and VM/bare-metal machines. We use different key provisioning mechanism for the two scenarios.
```

Comment:
provisioning mechanisms for each scenario.

Reply:
done